readme.md

```diff
@@ -70,12 +70,13 @@ DATABASE_URL=postgres://user:pass@host1:port/db1,mysql://user:pass@host2:port/db
 
 ### Uninstalling
 
-To remove the buildpack run the command.
+To remove the buildpack run the command and unset your VC_API_TOKEN.
```

Comment:
"... run the command and unset..." sounds like missing a command. What about "To remove the buildpack and unset your VC_API_TOKEN run the following commands:"